packages/lib/src/ui/blocks/Header/Header.tsx

```diff
@@ -29,22 +29,17 @@ const Navigation = styled.nav`
   box-sizing: border-box;
   display: flex;
   justify-content: space-between;
-  height: ${theme.navSize.vertical.height};
+  height: ${theme.size.base * 3.8}rem;
 
   @media ${min("l")} {
-    height: ${theme.navSize.horizontal.height};
+    height: ${theme.size.base * 3.3}rem;
   }
-  padding: ${theme.navSize.horizontal.padding};
 `
 
 const NavButton = styled(Button)`
   align-self: center;
-  padding: 0.4em 0.6em;
-  font-size: ${theme.navSize.vertical.buttonSize};
-
-  @media ${max("l")} {
-    display: unset;
-  }
+  padding: 0.65rem 1.1rem;
```

Comment:
Paddingy by možná měly být spíš v `em` aby byly relativní k velikosti písma přímo v něm. Ale nechám na tobě. Není to tak důležité.